docs/monorepo.md

```diff
@@ -63,21 +61,19 @@ The only exception are `devDependencies` which _must be declared in the wp-calyp
 	"bugs": {
 		"url": "https://github.com/Automattic/wp-calypso/issues"
 	},
-	"files": [
-		"dist",
-		"src"
-	],
+	"files": [ "dist", "src" ],
 	"scripts": {
 		"clean": "npx rimraf dist",
 		"prepublish": "npm run clean",
-		"prepare": "npx @automattic/calypso-build"
+		"prepare": "npx --package @automattic/calypso-build transpile"
```

Comment:
Needs updating to the ugly version?

Reply:
See convo starting at https://github.com/Automattic/wp-calypso/pull/32295#discussion_r276342009. Takeaway: Reverting the ugly version for now.